src/middlewared/middlewared/plugins/crypto.py

```diff
@@ -432,9 +432,16 @@ def get_x509_subject(self, obj):
             if 'subjectAltName' == ext.get_short_name().decode():
                 cert_info['san'] = [s.strip() for s in ext.__str__().split(',') if s]
 
-            cert_info['extensions'][
-                re.sub(r"^(\S)", lambda m: m.group(1).upper(), ext.get_short_name().decode())
-            ] = ext.__str__()
+            try:
+                cert_info['extensions'][
+                    re.sub(r"^(\S)", lambda m: m.group(1).upper(), ext.get_short_name().decode())
+                ] = ext.__str__()
+            except crypto.Error:
```

Comment:
Should we at lost log it to know something failed?

Reply:
Yup, also I try to at least retrieve the name if possible of the extension